tests/ui/perfherder/dropdown_menu_test.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import DropdownMenuItems from '../../../ui/shared/DropdownMenuItems';
+
+const selectedItem = 'fenix';
+const options = [
+  'mozilla-central',
+  'autoland',
+  'fenix',
+  'socorro',
+  'taskgraph',
+  'servo-master',
+  'ash',
+];
+const pinned = ['fenix', 'autoland', 'tunafish', 'socorro'];
+const updateData = () => {};
+
+const dropdownMenuItems = () =>
```

Comment:
We're testing the repository dropdown menu specifically, not the generic `DropdownMenuItems` component (which doesn't do any kind of option sorting).

I think we should rename the `dropdownMenuItems` to something that reflects this.